composer.json

```diff
@@ -17,7 +17,8 @@
         }
     ],
     "require": {
-        "php": "^7.3",
+        "php": "^7.3 || ^8.0",
+        "illuminate/support": "^6.18.25 || ^7.19.1 || ^8.0",
```

Comment:
Would be cool to copy/past the code used `Reflector::getParameterClassName` so we don't depend on this.

Reply:
Yeh, I was just thinking to do this. I think that'll fix the composer lowest issues.